com.unity.render-pipelines.high-definition/Runtime/Material/Lit/LitTessellation.shader

```diff
@@ -804,6 +804,37 @@ Shader "HDRP/LitTessellation"
 
             ENDHLSL
         }
+
+        Pass
```

Comment:
still reamin LayerdLit, LayeredLitTessellation, Axf and shader graph, but I get first you validate with this